app/app/route.js

```diff
@@ -5,11 +5,16 @@ export default Ember.Route.extend({
     if (tokens.length === 0) {
       tokens.push(this.currentModel.get('handle'));
     }
-    var stack = this.currentModel.get('stack');
+
+    let stack = this.get('stack');
+    Ember.assert('App Route must have a stack', stack);
```

Comment:
Cool, I'll change this. This is a dev-only optimization because `Ember.assert` is stripped in production, right?